presto-hive/src/main/java/com/facebook/presto/hive/cache/alluxio/AlluxioCachingModule.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.cache.alluxio;
+
+import com.facebook.airlift.configuration.AbstractConfigurationAwareModule;
+import com.facebook.presto.hive.DynamicConfigurationProvider;
+import com.google.inject.Binder;
+import com.google.inject.Scopes;
+
+import static com.google.inject.multibindings.Multibinder.newSetBinder;
+
+public class AlluxioCachingModule
```

Comment:
We can make it a CacheModule for presto-hive. Put all cache-related binding to this module including alluxio and local-range

Reply:
Sounds good, I will try to see how this fits in.